plugin/server.py

```diff
@@ -60,6 +61,7 @@ def get_audio(self, media_dir, file_path):
             return
         self.send_response(200)
         self.send_header("Content-type", mime_type)
+        self.send_header('Content-length', str(os.stat(audio_file).st_size))
```

Comment:
Could you use double quotes for the strings here? Sorry we don't have a linter for this project setup yet...